data_structures/stacks/stack_with_two_queues.py

```diff
@@ -0,0 +1,68 @@
+from collections import deque
+from typing import Generic, TypeVar
+
+T = TypeVar("T")
+""" Implementing stack using two arrays"""
+
+
+class Stack(Generic[T]):  # Stack class to implement stack operations
+    def __init__(self) -> None:
+        self.insert_queue = deque()  # First Queue to be used for inserting
+        self.suffle_queue = deque()  # Second Queue to be used for suffling
+
+    def push(self, item: int) -> None:
+        self.insert_queue.append(item)  # Add items into the Queue
+        while self.suffle_queue:
+            self.insert_queue.append(
+                self.suffle_queue.popleft()
+            )  # Popping the elements
+
+        self.insert_queue, self.suffle_queue = self.suffle_queue, self.insert_queue
+
+    def pop(self) -> int:
+        if not (self.suffle_queue):  # if the stack  is empty
+            return None
+        return self.suffle_queue.popleft()  # if not empty pop
+
+    def top(self) -> int:
+        if not (self.suffle_queue):
+            return None
+        return self.suffle_queue[0]
+
+    def printing(self) -> None:
+        print(self.suffle_queue)
+
+    def size(self) -> int:
+        return len(self.suffle_queue)
+
+
+def test_stack() -> None:
+    s = Stack()  # Creating a stack in S
```

Comment:
Single-letter variable names makes code look like it was written in the 1970's.
```suggestion
    stack = Stack()  # Create a empty stack
```